Automattic/wp-calypso
# Performance Profiler: update history placeholder

Closes https://github.com/Automattic/dotcom-forge/issues/8998

## Proposed Changes

* Update the history chart placeholder

## Why are these changes being made?


https://github.com/Automattic/dotcom-forge/issues/8998

## Testing Instructions



* Start a new test for a site without CrUX data, or use this report URL:
```
/speed-test-tool?url=https%3A%2F%2Fgcsecseyfree.wpcomstaging.com%2F&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NzQ3NH0.3NqZhZmRj6_aW3TexJ_RO3CU8wiqoK2mz69314mvDrQ
```
* Check the history chart styling and the updated copy

| Before | After |
|--------|--------|
| <img width="1065" alt="image" src="https://github.com/user-attachments/assets/2e587a4b-478d-4c08-8820-9e83fa3cef1c"> | ![CleanShot 2024-09-11 at 11 43 26@2x](https://github.com/user-attachments/assets/9644f6f4-1bca-496c-9cf5-e6fe8659e60b) |

## Pre-merge Checklist
